src/main/java/standard/StudentManagement/StudentManagementApplication.java

```diff
@@ -19,27 +18,40 @@
 
 public class StudentManagementApplication {
 
-  private List<Map<String, String>> studentList = new ArrayList<>();
-
-  public StudentManagementApplication() {
-  }
+  @Autowired
+  private StudentRepository repository;
 
   public static void main(String[] args) {
     SpringApplication.run(StudentManagementApplication.class, args);
   }
 
-  @GetMapping("/studentInfo")
-  public List<Map<String, String>> getStudentInfo() {
-    return studentList;
+
+  @GetMapping("/student")
+  public String getStudent(@RequestParam String name) {
+    Student student = repository.searchByName(name);
+    return student.getName() + " " + student.getAge() + "歳" + " ID " + student.getId();
+  }
```

Comment:
レスポンスをStringでなくJSONになるようにしましょう。
```suggestion
  public Student getStudent(@RequestParam String name) {
    Student student = repository.searchByName(name);
    return student;
  }
```
にすればよいです。

Reply:
ほかのAPIも修正してみてほしいです！

Reply:
修正してみましたのであっているのか確認をお願いします！